src/table.ts

```diff
@@ -741,10 +741,6 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
       ranges.push({});
     }
 
-    if (options.filter) {
-      filter = Filter.parse(options.filter);
-    }
-
```

Comment:
This block along with the `if (filter)` line below is moved to `#readRowsReqOpts`.